app/assets/stylesheets/forever_style_guide/modules/_list.scss

```diff
@@ -1,11 +1,21 @@
 /* list
 ---------------------------------------------------------------------------- */
+$badge-width: 24px;
```

Comment:
Whoops, definitely not. Though i do think this is ultimately a local variable given the application here.